var/spack/repos/builtin/packages/r-tidyverse/package.py

```diff
@@ -0,0 +1,66 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RTidyverse(RPackage):
+    """The 'tidyverse' is a set of packages that work in harmony because they
+       share common data representations and 'API' design. This package is
+       designed to make it easy to install and load multiple 'tidyverse'
+       packages in a single step. Learn more about the 'tidyverse' at
+       <https://tidyverse.org>."""
```

Comment:
This URL might be causing the documentation tests to fail. You can probably remove the last sentence, or even the last 2 sentences.